x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlIndexTemplateRegistry.java

```diff
@@ -59,6 +58,9 @@
 
     private static final IndexTemplateConfig STATS_TEMPLATE = statsTemplate();
 
+    private static final LifecyclePolicyConfig ANOMALY_DETECTION_STATE_ILM_POLICY =
+        new LifecyclePolicyConfig("state_index_ilm_policy", ANOMALY_DETECTION_PATH + "state_index_ilm_policy.json");
```

Comment:
What is the scope of the name `state_index_ilm_policy` here?  Is it local to the ML plugin or is it global across the whole of Elasticsearch?  If it is global I think it should be changed to `ml_ state_index_ilm_policy`.

Reply:
You're right, ILM, SLM and Watcher use this format.
So I've made my name consistent with them.

Note that I purposely did not change the file name (`state_index_ilm_policy.json`) as I wanted it to be consistent with `state_index_template.json` we already have in the same directory.